Google.PowerShell/Common/AuthenticateWithSdkCredentialsExecutor.cs

```diff
@@ -36,13 +36,14 @@ public class AuthenticateWithSdkCredentialsExecutor : ICredential, IHttpExecuteI
         /// rather than launching gcloud again.
```

Comment:
In the class declaration, put the `: ICredential, IHttpExecuteInterceptor, ` etc. on a new line to avoid going over 120 chars?